src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -9,6 +11,10 @@ public class LottoTicket {
     /** 구매한 로또 번호 */
     private final List<Integer> numbers;
 
+    public LottoTicket(String[] numbers) {
+        this.numbers = Arrays.stream(numbers).map(Integer::parseInt).collect(Collectors.toList());
```

Comment:
```suggestion
this(Arrays.stream(numbers)
      .map(Integer::parseInt)
      .collect(Collectors.toList());
```
이제 String[] 타입으로도 로또티켓을 만들수 있군요
다음과 같이 작성하면 주생성자에서 관리가 가능하답니다.

Reply:
아 지난번에 알려주신게 이거군요 ! `생성자 this` 를 활용해볼게요 !